client/src/components/TestSuite/TestSuiteDetails/TestSuiteDetailsPanel.tsx

```diff
@@ -53,12 +50,12 @@ const TestSuiteDetailsPanel: FC<TestSuiteDetailsPanelProps> = ({
     ) : null;
 
   return (
-    <div className={styles.testSuiteDetailsPanel}>
+    <>
```

Comment:
I'm not familiar with this. Does this actually end up in the rendered page or is it just a way to group multiple components together?

Reply:
Just a grouping! The return requires a wrapper component but because there is no need for an actual container here we can use a `React.Fragment` component, the shorthand of which is just `<> </> `(https://reactjs.org/docs/fragments.html#short-syntax)